src/test/java/jewellery/inventory/unit/service/security/JwtTokenServiceTest.java

```diff
@@ -126,7 +126,7 @@ void isTokenValidWithExpiredTokenThrowsJwtExpiredException() {
   @Test
   void generateTokenWithNullUserDetailsThrowsIllegalArgumentException() {
     lenient().when(jwtUtils.getSigningKey()).thenReturn(key);
-    lenient().when(userDetails.getUsername()).thenReturn(USER_NAME);
+    lenient().when(userDetails.getUsername()).thenReturn(FIRST_NAME);
```

Comment:
Considering username is no longer unique, shouldn't we be using the email as a username in the UserDetails service?

Reply:
Yes, we are using the email, just the getter is getUsername. But it returns the email. I think I made it that way because User implements UserDetails where we have field Username and the @Getter annotation creates a getter and I Override it so it returns the email...

Reply:
Sure, I get that but in the test instead of email, we are stubbing it to return FIRST_NAME

Reply:
Oh, my bad. Actually the stubbing should return null since we test if exception is thrown when userDetails is null.. Fixed